plugins/module_utils/_netapp.py

```diff
@@ -384,8 +384,8 @@ def request(self, path, data=None, method='GET', headers=None, ignore_errors=Fal
             path = path[1:]
         request_url = self.url + self.DEFAULT_REST_API_PATH + path
 
-        if self.log_requests or True:
-            self.module.log(pformat(dict(url=request_url, data=data, method=method)))
+        # if self.log_requests:
+        self.module.log(pformat(dict(url=request_url, data=data, method=method)))
```

Comment:
This looks distinctly like someone's debugging leaked into the code

```suggestion
        if self.log_requests:
            self.module.log(pformat(dict(url=request_url, data=data, method=method)))
```

Reply:
True, but changing that would break backwards compatibility. Since this is deprecated/dead code, I don't want to make unnecessary changes to it (even though I think you are right).

I'll add a follow-up PR to remove that module anyway, since it's no longer used. But since it's still used in stable-2 (and this PR will be backported there as well), I don't want to do that in this PR :)